tests/answers-umd.js

```diff
@@ -0,0 +1,63 @@
+import ANSWERS from '../src/answers-umd';
+
+let windowSpy;
+
+describe('ANSWERS instance integration testing', () => {
+  beforeEach(() => {
+    windowSpy = jest.spyOn(window, 'window', 'get');
```

Comment:
is this needed so we can reset the window after each test?

Reply:
ya, it's setup this way so that on each test we can have a fresh window to work with